pkg/skaffold/inspect/namespaces/list.go

```diff
@@ -58,10 +57,6 @@ func PrintNamespacesList(ctx context.Context, out io.Writer, manifestFile string
 		if len(resourceYAML) == 0 {
 			continue
 		}
-		// - obj is the API object (e.g., Deployment)
```

Comment:
why are we deleting this comment?

Reply:
I thought it wasn't necessary initially.  I have added back now